### ISSUE TITLE
fails to install gopls due to missing xcrun (xcode) installation

### ISSUE THREAD

**Issue by poembro** [golang/vscode-go#3103]
```
1 tools failed to install.

gopls: failed to install gopls(golang.org/x/tools/gopls@latest): Error: Command failed: /usr/local/go/bin/go install -v golang.org/x/tools/gopls@latest
runtime/cgo
# runtime/cgo
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun

```

**Comment by hyangah** [comment]
The next version of gopls (v0.15, to be released next year) will not have the cgo dependency.
Actually, the cgo dependency is already removed in the dev version of gopls. 
Can you try the unstable version? 

```
git clone https://go.googlesource.com/tools
cd tools/gopls
go install
```

**Comment by ansaba** [comment]
Closing it. It will be fixed with next gopls v0.15.0
